Jenkinsfile

```diff
@@ -1,22 +0,0 @@
-pipeline {
-  agent any
-
-  options {
-    ansiColor('xterm')
-    disableConcurrentBuilds()
-    buildDiscarder(logRotator(numToKeepStr: '10')) // only keep artifacts of the last 10 builds
-  }
-
-  stages {
-    stage('Test gem') {
-      steps {
-        sh "bnw_runner ./_pipeline/step_test_gem.sh"
-      }
-    }
```

Comment:
Is this step cover somewhere now?

Reply:
https://github.com/simplybusiness/flip_fab/pull/41#discussion_r931023089